v2/cf-nodejs-logging-support/src/lib/config/config-validator.ts

```diff
@@ -0,0 +1,17 @@
+import ConfigShema from "./config-schema.json";
+import Ajv from "ajv";
+
+export default class ConfigValidator {
+
+    private static ajv = new Ajv();
+
+    static isValid(config: any): boolean | Error {
+        const validate = ConfigValidator.ajv.compile(ConfigShema);
+        const valid = validate(config);
+        if (!valid) {
+            const error = JSON.stringify(validate.errors);
+            throw new Error("Something in the configuration file is not valid. Please check error:" + error);
+        }
+        return true;
+    }
```

Comment:
The method returns `true` or thows an error. Based on the naming one could assume that it indicates invalid configurations by returning `false`. Could we do the following:
Always return `true` or `false`. And if invalid (=`false`), we also return the error as secondary return parameter: [boolean, error]. Throwing it would happen in config.ts.